manticore/ethereum/plugins.py

```diff
@@ -84,9 +84,10 @@ def will_open_transaction_callback(self, state, tx):
             else:
                 # Avoid all not selected hashes
                 for func_hsh in md.function_selectors:
+                    constraint = True
                     if func_hsh in selected_functions:
-                        constraint = tx.data[:4] != func_hsh
-                        state.constrain(constraint)
+                        constraint = Operators.AND(tx.data[:4] != func_hsh, constraint)
```

Comment:
Why the `AND True` here? Does this affect the plugin enable/disable operation?

Reply:
No, it does not affect the main PR goal. 
This builds a single constraint and add that to the constraintSet rather than add one for each function id this matches.

Reply:
I could be misreading this, but shouldn't `constraint=True` be outside the loop then?

Reply:
YES!